src/Function/Relation/Binary/Equality.agda

```diff
@@ -0,0 +1,35 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Function Equality setoid
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Function.Relation.Binary.Equality where
```

Comment:
So it would be great if we could structure this better than the old hierarchy. In particular this should be `Equality.Setoid` so we have room for the propositional version as well.

Reply:
LOL, I named this according to your own suggestion in one of the issues! But sure, will do.